src/components/color_picker/color_picker.xml

```diff
@@ -71,7 +71,7 @@
               t-on-click.stop=""
               t-att-value="state.customHexColor"
               t-on-input="setHexColor"
-              maxlength="7"
+              maxlength="8"
```

Comment:
nitpick: maxLength is kinda useless if we change the value at each `t-on-input` anyway to slice it, no ?